src/app/containers/PageHandlers/withContexts.jsx

```diff
@@ -46,12 +48,14 @@ const WithContexts = Component => {
     previousPath: string,
     service: string.isRequired,
     dials: objectOf(bool).isRequired,
+    currentPath: string,
```

Comment:
We should always have a `currentPath` right? I don't see why this can't be a `required` prop.